chcemvediet/apps/inforequests/forms/obligee_action.py

```diff
@@ -355,28 +401,30 @@ def post_transition(self):
         res = super(IsItAppealDecision, self).post_transition()
 
         if not self.is_valid():
+            #todo: check this
             res.next = ContainsAppealInfo
         elif self.cleaned_data[u'is_decision']:
             res.next = ContainsAppealInfo
         else:
-            res.next = NotCategorized
+            res.next = CanAddDisclosureRefusalAdvancementExtension
 
         return res
 
-class CanAddAppealDecision(ObligeeActionStep):
+class CanAddRemandmentAffirmationReversion(ObligeeActionStep):
 
     def pre_transition(self):
-        res = super(CanAddAppealDecision, self).pre_transition()
+        res = super(CanAddRemandmentAffirmationReversion, self).pre_transition()
         branch = self.wizard.values.get(u'branch', None)
 
         if self.wizard.email:
-            res.next = NotCategorized
+            res.next = CanAddDisclosureRefusalAdvancementExtension
         elif not branch:
             res.next = IsItAppealDecision
         elif branch.can_add_remandment:
+            #todo: check this
```

Comment:
@martinmacko47 Pripada mi to divne ked podmienka `elif not branch` a `elif branch.can_add_remandment` maju rovnaky vysledok. Ak neexistuje `branch`, nemal by wizard skocit na dalsiu cast?
```
if not branch:
    res.next = CanAddDisclosureRefusalAdvancementExtension
```

Reply:
@viliambalaz Ak si spravne pamatam, podmienka `elif not branch` by mala byt pravdiva, iba ked je este v prvom kroku a ziadnu branchu este nevybral. Avsak aj vtedy potrebujeme odsimulovat ktory wizardu dopredu a spocitat, kolko krokov wizard ma. Preto v takych nevalidnych pripadoch vzdy vyberame vetvu, ktora je pravdepodobnejsia, alebo dlhsia, aby spocitany pocet krokov wizardu bol realisticky.

Kedze ked brancha vybrana este nie je, tak nevieme povedat, ci wizard skoci do "pre appeal" alebo do "post appeal" vetvy, tak je v principe jedno, ktoru vetvu vyberieme. Najlepsie asi vybrat tu dhsiu. Teda v `CanAddRemandmentAffirmationReversion` vyberme vetvu "no" a v `CanAddDisclosureRefusalAdvancementExtension` vetvu "yes".

Vo vseobecnosti, vsetky vybery vetviev v nevalidnych stavoch, by mali viest k co najvaciemu poctu krokov wizardu. Nech user vidi, kolko najviac krokov wizard bude mat.